spec/features/charge_and_pass_libraries_spec.rb

```diff
@@ -27,9 +27,9 @@
   context 'plate with no submissions to be made' do
     before { create :purpose_config, uuid: 'example-purpose-uuid' }
 
-    let(:labware_barcode) { example_plate_v2.labware_barcode.machine }
+    let(:plate_barcode) { plate.labware_barcode.machine }
```

Comment:
I thought it was `.machine_barcode`, but maybe its `.machine` 😃

Reply:
I think it is `machine_barcode` on Sequencescape but then it has become `machine` by the time it's gone through various layers into Limber.

```
# Wraps labware barcodes to assist with rendering and to provide a cleaner interface
class LabwareBarcode
  attr_reader :human, :machine, :ean13

  def initialize(human:, machine:, ean13: nil)
    @human = human
    @machine = machine
    @ean13 = ean13
  end
```